jobs/pacman-rule-engine-2.0/src/main/java/com/tmobile/pacman/util/NotificationUtils.java

```diff
@@ -57,7 +57,7 @@ public static void triggerNotificationsForViolations(List<Annotation> annotation
             }
         }
         catch (Exception e) {
-            LOGGER.error("Error triggering lambda function url, notification request not sent. Error - {}",e.getMessage());
+            LOGGER.error("Error occurred in job with policyUUID: "+annotations.get(0).get(POLICY_NAME)+" Error message - Error triggering lambda function url, notification request not sent, "+e.getMessage());
```

Comment:
consider extracting this to a constant and standardizing the errors per same comments as for the collectors